app/src/main/java/us/kikin/apps/android/bgplayer/ui/show/ShowListFragment.kt

```diff
@@ -8,31 +8,29 @@ import android.widget.Toast
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
 import androidx.navigation.fragment.findNavController
-import androidx.recyclerview.widget.RecyclerView
 import dagger.hilt.android.AndroidEntryPoint
-import us.kikin.apps.android.bgplayer.R
+import us.kikin.apps.android.bgplayer.databinding.FragmentShowBinding
 import us.kikin.apps.android.bgplayer.models.VideoShowModel
 import us.kikin.apps.android.bgplayer.ui.videos.VideoItemClickListener
 
 @AndroidEntryPoint
 class ShowListFragment : Fragment(), VideoItemClickListener, ShowItemClickListener {
 
+    private var _binding: FragmentShowBinding? = null
+    private val binding get() = _binding!!
```

Comment:
I haven't used ViewBindings before, but isn't there a cleaner way to do this than `!!`?

Reply:
Again, this is taken from official docs, but technically you could use `?: throw SomeException` instead, just to be slightly more explicit

Reply:
Yeah I don't like `!!` because it's reintroducing NPEs in Kotlin. `requireNotNull` would be better (it throws an `IllegalStateException`, which is a clearer intent)

I am surprised they don't have a better solution for Kotlin code